pom.xml

```diff
@@ -146,7 +146,13 @@
 				<artifactId>spring-aot</artifactId>
 				<version>${spring-native.version}</version>
 			</dependency>
-
+			<dependency>
+				<groupId>org.junit</groupId>
+				<artifactId>junit-bom</artifactId>
```

Comment:
I think this is already imported via:
`spring-cloud-build-dependencies` -> `spring-boot-dependencies`

Reply:
That's right; this block can be removed.